src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,149 @@ Master Pre-Releases
+  `PR #7794 <https://github.com/pantsbuild/pants/pull/7794>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Remove an old redundant .isort.cfg. (#7859)
+  `PR #7859 <https://github.com/pantsbuild/pants/pull/7859>`_
+
+* Fix `DistributionLocator` dependency declarations. (#7875)
+  `PR #7875 <https://github.com/pantsbuild/pants/pull/7875>`_
+
+* Stabilize all tests of NailgunTask subclasses. (#7866)
+  `PR #7866 <https://github.com/pantsbuild/pants/pull/7866>`_
+
+* NailgunTasks execute java with consistent cwd. (#7872)
+  `PR #7872 <https://github.com/pantsbuild/pants/pull/7872>`_
+
+* add debug log for jvm target compile time ; fix execution graph display (#7873)
+  `PR #7873 <https://github.com/pantsbuild/pants/pull/7873>`_
+
+* Use argfile support in zinc and rsc (#7868)
+  `PR #7868 <https://github.com/pantsbuild/pants/pull/7868>`_
+
+* Create a separate crate for workunit_store (#7855)
+  `PR #7855 <https://github.com/pantsbuild/pants/pull/7855>`_
+
+* rsc re-uses DirectoryDigest from process execution rather than re-snapshotting (#7861)
+  `PR #7861 <https://github.com/pantsbuild/pants/pull/7861>`_
+
+* Extract out `resolve_requirements` V2 rule for creating PEXes with requirements (#7846)
+  `Issue #7795 <https://github.com/pantsbuild/pants/issues/7795>`_
+  `PR #7846 <https://github.com/pantsbuild/pants/pull/7846>`_
+
+* Split up CI unit tests into two distinct shards (#7867)
+  `PR #7867 <https://github.com/pantsbuild/pants/pull/7867>`_
+
+* Bump to latest zinc in order to consume zinc analysis fix (#7854)
+  `PR #7854 <https://github.com/pantsbuild/pants/pull/7854>`_
+
+* Port `ci.sh` to Python 3 for more descriptive CLI flags and less duplication (#7849)
+  `PR #7849 <https://github.com/pantsbuild/pants/pull/7849>`_
+
+* rsc doesn't re-snapshot jars produced by zinc (#7858)
+  `PR #7858 <https://github.com/pantsbuild/pants/pull/7858>`_
+
+* add java_sources to scala_library() snapshot (#7840)
+  `PR #7840 <https://github.com/pantsbuild/pants/pull/7840>`_
+
+* Apply isort fixes (#7857)
+  `PR #7857 <https://github.com/pantsbuild/pants/pull/7857>`_
+
+* Locate the classes directory in order to relativize classnames (#7853)
+  `PR #7853 <https://github.com/pantsbuild/pants/pull/7853>`_
+
+* Move V2 test runner integration test into proper location of `backend/python` folder (#7847)
+  `PR #7847 <https://github.com/pantsbuild/pants/pull/7847>`_
+
+* Run contrib tests with unit tests and integration tests for fewer CI delays (#7709)
+  `PR #7709 <https://github.com/pantsbuild/pants/pull/7709>`_
+
+* No longer default to saving non-deterministic run information to PEX in `./pants binary` (#7843)
+  `PR #7843 <https://github.com/pantsbuild/pants/pull/7843>`_
+
+* Cleanup unnecessary code from #7776 for `./pants binary` interpreter constraints (#7842)
+  `PR #7776 <https://github.com/pantsbuild/pants/pull/7776>`_
```

Comment:
It seems like this should have itself linked as well:
```
`PR #7842 <https://github.com/pantsbuild/pants/pull/7842>`_
```

Reply:
Ah, good catch, this link is to a referenced commit rather than the PR.

There are times the changelog helper appears to match more greedily than I expect. Is the preference to include links to all referenced PRs (so, in this case both #776 and #7842)?

For now I am going to link just the true PR - happy to restore the additional links if anyone thinks that best.